mne_bids/config.py

```diff
@@ -0,0 +1,3 @@
+"""Configuration values for MNE-BIDS."""
+
+BIDS_VERSION = "1.0.2 (draft)"
```

Comment:
necessary in new file?

Reply:
or let it be ...

Reply:
I couldn't figure out a way to import this from one of the other modules without creating a circular import :-/

Reply:
ah yes, makes sense